### PR TITLE
Prod ess

### LINKED ISSUE #56: Add CEC Data to ProdEnergyStorageSystem
[Energy_Storage_System_List_Data_ADA (1).xlsx](https://github.com/Open-Orange-Button/Orange-Button-Taxonomy/files/8612884/Energy_Storage_System_List_Data_ADA.1.xlsx)

This sheet looks similar to the CEC inverter list, and I expect that we'll be able to reuse SOME of the objects, but there are some ESS specific certifications.

### PR DESCRIPTION
**Pull request by blizita**
Closes #56, #39
Added CEC info to ProdESS (and a couple things to ProdInverter)

### CODE REVIEW

**Comment by jrippingale** [comment]
I think we need to update the What's New file to reference Pull Request 69 rather than the Git Isssue numbers 56 and 39.  I need Cliff / Kevin to confirm though.

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -10921,6 +10939,57 @@
...
+      },
+      "IsPVInputCapable": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementBoolean"
+          },
+          {
+            "type": "object",
+            "description": "Is this device capable of accepting PV input natively (without seperate charge controller)",
+            "x-ob-item-type": "stdi:booleanItemType",
```
Comment (reepoi) [inline Master-OB-OpenAPI.json:10968]:
```suggestion
            "x-ob-item-type": "BooleanItemType",
```

File: Master-OB-OpenAPI.json [hunk 2]
```diff
@@ -10921,6 +10939,57 @@
...
+      },
+      "EfficiencyRoundTrip": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "Percent of energy retained during full charge/discharge cycle",
+            "x-ob-item-type": "percentItemType",
```
Comment (reepoi) [inline Master-OB-OpenAPI.json:10985]:
```suggestion
            "x-ob-item-type": "PercentItemType",
```

**Comment by cwhanse** [comment]
Validation issues will be fixed in #67

**Pull request merged**